app/models/thesis.rb

```diff
@@ -55,6 +55,8 @@ class Thesis < ApplicationRecord
   validates :degrees, presence:
     { message: VALIDATION_MSGS[:degrees] }
 
+  # validates :files, presence: true
```

Comment:
Note. I put this here as an example of what you can now do with ActiveStorage in Rails 6 but that breaks a ton of tests that didn't consider this requirement of the model as they were built without it. I have confirmed even on the Rails 5.2 branch that if you just disable the JavaScript check in the uploader that ensures a file is attached it will submit it so this validation has never happened at the model level. We should remove this before merging but I wanted to make it easy for a reviewer to uncomment this locally to see what's going on if they wanted.

The new ETD features we are building will require this validation to be removed even if we did make it work properly now so it is legit not worth our time to fix this now.